2-Authorization-I/1-call-graph/SPA/src/graph.js

```diff
@@ -0,0 +1,109 @@
+import { InteractionRequiredAuthError, InteractionType } from '@azure/msal-browser';
```

Comment:
See https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/FAQ.md#what-can-i-do-outside-of-azuremsal-react-context

Can we instead make this a React component, so that it can subscribe to msal-react context and check for interaction status before making a token request?

Reply:
alternatively, it could go into a hook as well